rhm.js.org

gsongsong opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 reviewed (approved):
✅ Commit valid, CNAME present, site appears to have content \*

\* based off https://github.com/gsongsong/rhm/tree/master/docs

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to fix some 404 for your asset paths.
I'll merge after we have "contact". Please keep this open.

gsongsong commented:
> 
> 
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to fix some 404 for your asset paths.
> I'll merge after we have "contact". Please keep this open.

Thanks! I fixed 404 issues by adjusting base URL from `github.io` to `js.org`! (Of course no more PR is required)

indus commented:
Welcome https://rhm.js.org
